actions/AuthAction.js

```diff
@@ -142,44 +140,34 @@ async function checkForEarlyProductAccess(toggle, sdkKey, isInternal, org) {
   });
 }
 
-function asAuthAction(action) {
+function asAuthAction(action, isAuthorize = true) {
```

Comment:
To improve readability, I'd name the second param `isAuthorized` or `shouldAuthorize`

Reply:
Renamed to `shouldAuthorize`, 👍